CHANGELOG.rst

```diff
@@ -5,6 +5,7 @@ ChangeLog
 master (unreleased)
 ===================
 
+- Allow POST method for form validation endpoint.
```

Comment:
FYI, there's a ``deprecations.rst`` document in the ``docs/`` directory. I think we should do the following:

1. Add a deprecation note here, in the Changelog, to inform the reader that this version (probably 1.3.0) will be the last to support the GET verb for the validation endpoint.
2. When we'll do the release, we'll add this point to the ``deprecations.rst`` document with the correct version number.

Reply:
The changelog has been updated :)